docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,465 @@
+* [Cross namespace support for service invocation](https://github.com/dapr/docs/tree/master/howto/invoke-and-discover-services#namespaces)
+* New components: [RethinkDB binding](https://github.com/dapr/docs/blob/master/howto/setup-state-store/setup-rethinkdb.md), [Open Policy Agent middleware](https://github.com/dapr/docs/blob/master/howto/policies-with-opa/README.md), [Bi-directional PostgresSQL binding](https://github.com/dapr/docs/blob/master/reference/specs/bindings/postgres.md)
+
+
+If you're new to Dapr, visit the [getting started](https://github.com/dapr/docs/tree/master/getting-started) page and familiarize yourself with Dapr.
+
+Docs have been updated with all the new features and changes of this release. To get started with new capabilities introduced in this release, go to the [Concepts](https://github.com/dapr/docs/tree/master/concepts) and the [How To section](https://github.com/dapr/docs/tree/master/howto#how-tos).
+
+> **Note: This release contains a few [breaking changes](#breaking-changes).**
+
+See [this](#upgrading-to-dapr-0.11.0) section on upgrading Dapr to version 0.11.0.
+
+## New in this release
+
+### Dapr Runtime
+
+- **Refactored** code style, remove unnecessary code, use defer for unlock operation (Thanks @skyao)
+
+- **Refactored** to use github.com/pkg/errors instead of fmt.errorf (Thanks @jigargandhi and @ShaileshSurya)
+
+- **Added** code to increase Dapr logging verbosity from app -> sidecar
+
+- **Added** optional metadata field to the State APIs
+
+- **Removed** non transactional state APIs from actors
+
+- **Added** error field to Bulk Get API
+
+- **Fixed** empty fields in Bulk Get API JSON response
+
+- **Changed** perf test payload size
+
+- **Added** service invocation REST API Swagger docs (Thanks @dancadana)
+
+- **Fixed** actor returns 200 on GET if key is not present
+
+- **Added** requests authentication from api-server in sidecar injector
+
+- **Fixed** 'host removed' log prints multiple times per second
+
+- **Fixed** response handling from pub/sub message delivery on gRPC
+
+- **Added** E2E tests for actor state scenario
+
+- **Updated** E2E test components to fetch values from secrets (Thanks @yittg)
+
+- **Added** support for cross namespace service discovery in service invocation
+
+- **Added** decision record for parity between GetState and SaveState API
+
+- **Fixed** return 400 on actor not found 
+
+- **Updated** speed of Windows CI E2E tests
+
+- **Refactored** component processing during runtime init (Thanks @yittg)
+
+- **Fixed** actor E2E tests intermittent failure
+
+- **Added** log for a loaded component in runtime
+
+- **Changed** NATS server to NATS streaming for pub/sub (Thanks @abserari)
+
+- **Added** declarative mechanism to create topic subscription
+
+- **Fixed** sidecar injector leaks private info in logs of Kubernetes admission response  **(Needs separate note)**
```

Comment:
**Fixed** sidecar injector logging admission response (in previous release, under certain condition when running in Kubernetes, this could have included potentially private information).